charts/akeyless-secure-remote-access/Chart.yaml

```diff
@@ -14,11 +14,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 
-version: 0.24.4
+version: 0.24.5
 
 appVersion: v0.20.2_0.15.7
```

Comment:
change to v0.20.2_0.15.7_1.2.5

Reply:
done

Reply:
@michelsk have you updated the ZTP deployment from frontent repo to updated the app version each deployment?

Reply:
we have a jira ticket for that